create-issue/action.yml

```
@@ -0,0 +1,30 @@
+name: Create issue
+description: Create an issue in a repository
+author: ZhongRuoyu
+branding:
+  icon: alert-circle
+  color: blue
+inputs:
+  token:
+    description: GitHub token
+    required: false
+    default: ${{github.token}}
+  repository:
+    description: Repository to send the issue to
+    required: false
+    default: ${{github.repository}}
```

Would be nice to have a linter for this
```suggestion
    default: ${{ github.token }}
  repository:
    description: Repository to send the issue to
    required: false
    default: ${{ github.repository }}
```